util/src/validate.ts

```diff
@@ -107,13 +106,11 @@ function readInput(toolOutput: LSIF.Element[]): void {
 				}
 
 				visited[inV.toString()] = visited[outV.toString()] = true;
-			}
-			if (Edge.is11(edge)) {
+			};
+			if (LSIF.Edge.is11(edge)) {
```

Comment:
Would it make more sense to add a helper method that just enumerates the inVs even if it is a 1:1 edge instead of having 1:1 1:M checks in many places? Then you can treat all edges as 1:M to simplify things

Reply:
I love this suggestion, I will work on that. Thanks!